protecode/scanning_util.py

```diff
@@ -234,30 +235,33 @@ def _metadata(
 
     def upload_container_image_group(
         self,
-        resource_group: ResourceGroup,
+        upload_elements: typing.List[UploadElement],
```

Comment:
do you actually need a list? if so, you could use the shorthand form `list[T]`. if not, rather use `typing.Iterable`